solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -345,6 +345,36 @@ describe("audius-data", () => {
     await confirmLogInTransaction(provider, tx3, updatedTrackMetadata);
   });
 
+  it("Creating User!", async () => {
+    let {
```

Comment:
failure cases? ex. create a user that already exists, attempt to create a user that was initialized by admin, et.c

Reply:
Added these tests